hack/Dockerfile.debug

```diff
@@ -1,12 +1,14 @@
 FROM registry.access.redhat.com/ubi9/ubi
-RUN yum install -y https://github.com/frobware/haproxy-builds/raw/master/rhaos-4.16-rhel-9/haproxy28-2.8.5-1.rhaos4.16.el9.x86_64.rpm
-RUN haproxy -vv
+
+# For example: https://github.com/frobware/haproxy-builds/raw/master/rhaos-4.17-rhel-9/haproxy28-2.8.10-1.rhaos4.17.el9.x86_64.rpm
```

Comment:
Is there an example available in brew or another place not referencing your repo?

Reply:
HAProxy RPM artefacts are available from here: https://ocp-artifacts.hosts.prod.psi.rdu2.redhat.com/pub/RHOCP/plashets/4.18/stream/el9/latest/x86_64/os/Packages/haproxy-2.8.10-1.rhaos4.17.el9__x86_64/. But you need to be on the VPN to access those. I can add this link but you would still need to copy the RPM somewhere that is accessible if your cluster is not on the VPN (which seems likely). Does that help?

Reply:
> Is there an example available in brew or another place not referencing your repo?

Updated to use Red Hat location in ba8f66c29cc80b1cd59d93ca9f12d2d0f0b8439e.